revolv/templates/ambassador/dashboard.html

```diff
@@ -3,103 +3,130 @@
 
```

Comment:
There's a LOT of repeated html in this file that was also in the other dashboard files, so they should all inherit from a parent template